app/templates/app/delete_installment.html

```diff
@@ -0,0 +1,43 @@
+{% extends 'base.html' %}
+{% load bootstrap4 %}
+{% bootstrap_css %}
+{% load static %}
+{% load staticfiles %}
+{% load i18n %}
+{% load render_table from django_tables2 %}
+{% block content %}
+<html>
+    <body>
+    <div class='m-5 p-3'>
+        <form method="post">{% csrf_token %}
+        <div class="row justify-content-center m-3">
+                    <div class="card" style="width: 30rem;">
```

Comment:
Could you put this in a css related file? `style="width: 30rem;"` It isn't a good practice https://stackoverflow.com/questions/2612483/whats-so-bad-about-in-line-css